public/walkthroughs/1/walkthrough.json

```diff
@@ -0,0 +1,6 @@
+{
+  "dependencies": {
+    "repos": [],
+    "services": []
```

Comment:
We could already add the WT1 services here. Should be the same as here: https://github.com/integr8ly/tutorial-web-app/blob/add-own-walkthroughs/public/walkthroughs/my-custom-walkthrough/walkthrough.adoc#a-task-resource, so: spring-boot-rest-http-crud and nodejs-messaging-work-queue-frontend.

The format should problably be:

```
{
  "dependencies": {
    "repos": [],
    "serviceInstances": [{
      "metadata": { "name": "spring-boot-rest-http-crud" },
      "spec": {
        "clusterServiceClassExternalName": "spring-boot-rest-http-crud",
        "clusterServicePlanExternalName": "default"
      }
    },
    ...
    ]
  }
}
```

Does that look right @aidenkeating ?

Reply:
Yes that looks correct.

It'd be:

```
{
  "dependencies": {
    "repos": [],
    "serviceInstances": [{
      "metadata": { "name": "spring-boot-rest-http-crud" },
      "spec": {
        "clusterServiceClassExternalName": "spring-boot-rest-http-crud",
        "clusterServicePlanExternalName": "default"
      }
    }, {
      "metadata": { "name": "nodejs-messaging-work-queue-frontend" },
      "spec": {
        "clusterServiceClassExternalName": "nodejs-messaging-work-queue-frontend",
        "clusterServicePlanExternalName": "default"
      }
    }]
  }
}
```